client/script.js

```diff
@@ -94,7 +94,7 @@ function handleSubscription(subscription) {
   if (payment_intent) {
     const { client_secret, status } = payment_intent;
 
-    if (status === 'requires_action' || status === 'requires_payment_method') {
+    if (status === 'requires_action') {
```

Comment:
The else block (https://github.com/stripe-samples/set-up-subscriptions/pull/20/files#diff-484245f14c67ce4fc2d956536e096dc0R110) now handles also the payment error case, right? So we should probably update the content on line 110 to reflect that.